asm/bss_8001C4D0.s

```diff
@@ -67,7 +67,7 @@ glabel D_80164AD0
 glabel gMatrixEffectCount
 .skip 20
 
-glabel playerPalette
+glabel gPlayerPalette
```

Comment:
Does this point to a single palette? Or to all the palettes?

gPlayerPalettes if it points to more than one.